util/parsenum.h

```diff
@@ -18,13 +18,13 @@
 #define PARSENUM(x, s, min, max)					\
 	(								\
 		errno = 0,						\
-		(((x) = 1, (x) /= 2) != 0) ?				\
-			((x) = parsenum_float((s), (min), (max))) :	\
-		(((x) = -1) <= 0) ?					\
-			((x) = parsenum_signed((s), (min), (max))) :	\
+		(((*x) = 1, (*x) /= 2) != 0) ?				\
```

Comment:
why on earth would somebody try testing `&foo + 5`, i don't even.

C can be amazingly horrifying.  :)